packages/contracts-bedrock/tasks/genesis-l2.ts

```diff
@@ -46,6 +58,120 @@ const getStorageLayout = async (
   throw new Error(`Cannot locate storageLayout for ${name}`)
 }
 
+// Find the contract and source from the build info
+const findContractAndSource = (name: string, buildInfo: BuildInfo) => {
+  const sources = buildInfo.output.sources
+  const contracts = buildInfo.output.contracts
+
+  const compilerOutputContracts: CompilerOutputContract[] = []
+  for (const [contractName, contract] of Object.entries(contracts)) {
+    if (path.basename(contractName, '.sol') === name) {
+      compilerOutputContracts.push(contract[name])
+    }
+  }
+  if (compilerOutputContracts.length !== 1) {
+    console.log(`Unexpected number of contracts for ${name}`)
```

Comment:
This is kind of a confusing error message if the number is 0.

Reply:
You are right, this should throw if none are found

Reply:
Pushed a fix for this